src/main/java/com/mattbozelka/repository/DatabaseConnection.java

```diff
@@ -0,0 +1,231 @@
+package com.mattbozelka.repository;
+
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.mattbozelka.model.*;
+
+
+public class DatabaseConnection {
+	public static boolean driverLoaded = false;
+	public static boolean databaseConnected = false;
+	public static Connection conn;
+	
+	public DatabaseConnection(){
+		if (!driverIsLoaded()){loadDriver();}
+		if (!databaseIsConnected()){connectToDatabase();}
+	}
+	
+	 public static void loadDriver() {
+        try {
+            // The newInstance() call is a work around for some
+            // broken Java implementations
+        	//com.mysql.jdbc.Driver
+            Class.forName("com.mysql.jdbc.Driver").newInstance();
+            driverLoaded=true;
+        } catch (Exception ex) {
+            // handle the error
+        	
+        	//TODO: delete after testing
+        	System.out.println("Error loading driver.");
+        	System.out.println(ex.toString());
+        	
+        	driverLoaded=false;
+        }
+    }
+	 
+	public void connectToDatabase() {
+		
+			try {
+
+				conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/cleanupstars","root","password1");
```

Comment:
We'll want to figure out how to utilize username/passwords on the server.
@captainMattador had some ideas for this.